### ISSUE TITLE
release 2.2.0

### ISSUE THREAD

**Issue by linuxmaniac** [sipcapture/sipgrep#44]
Can you, please, release 2.2.0 version so I can package it for Debian? 

sipgrep is removed from testing right now due to: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=999961

**Comment by adubovikov** [comment]
sure, let me do it right now

**Comment by adubovikov** [comment]
done

**Comment by linuxmaniac** [comment]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Format: 1.8
Date: Mon, 25 Mar 2024 09:52:16 +0100
Source: sipgrep
Architecture: source
Version: 2.2.0-1
Distribution: unstable
Urgency: medium
Maintainer: Debian VoIP Team [<pkg-voip-maintainers@lists.alioth.debian.org>](mailto:pkg-voip-maintainers@lists.alioth.debian.org)
Changed-By: Victor Seva [<vseva@debian.org>](mailto:vseva@debian.org)
Closes: 999961
Changes:
 sipgrep (2.2.0-1) unstable; urgency=medium
 .
   * New upstream version 2.2.0
   * remove already applied patches
   * debhelper-compat 13. wrap-and-sort -sat
   * build-depends pcre2 (Closes: #999961)
   * update Vcs-* URLs
   * install manpage
   * Stantads-Version 4.6.2, no changes needed. Use my debian account
   * update copyright path for uthash.h
Checksums-Sha1:
 a99acf4325edf50507f3b6924462cd024af613f0 1382 sipgrep_2.2.0-1.dsc
 572b1e4c4226c81a4a54575b851867bf8342812d 284625 sipgrep_2.2.0.orig.tar.gz
 66d56b9e46f3235a101df4e9be69279b83e49ed2 2416 sipgrep_2.2.0-1.debian.tar.xz
 c3b8a1c599c1d43ca94f7b7d8ecaf067bae95e9e 6376 sipgrep_2.2.0-1_amd64.buildinfo
Checksums-Sha256:
 b0066d65ad461388d3e68756b45178bb31d8cb20f0329cd116c572c93250515c 1382 sipgrep_2.2.0-1.dsc
 ee16c1efde73c8faa591f8d84b697b629fabf43806fb292893a3ba3f8cc9a290 284625 sipgrep_2.2.0.orig.tar.gz
 70099a217f84233a290a11cede876c29abec8ab3ff038cc9b83c87bd5b4415ee 2416 sipgrep_2.2.0-1.debian.tar.xz
 3bd16c69215a2d04584c2928371702f55a5372e737983cd1d286db8a214ae95c 6376 sipgrep_2.2.0-1_amd64.buildinfo
Files:
 da17febb3e663a915483e12ca8fa5517 1382 net optional sipgrep_2.2.0-1.dsc
 e94fcb80455337713695e31fc03242f1 284625 net optional sipgrep_2.2.0.orig.tar.gz
 c351fac55a3dc002c50132ccdbf83ab9 2416 net optional sipgrep_2.2.0-1.debian.tar.xz
 22e203ee60a4290598b36185c6d59ee0 6376 net optional sipgrep_2.2.0-1_amd64.buildinfo

-----BEGIN PGP SIGNATURE-----

iIcEARYKAC8WIQQq6AO8RS0zF4SC1vh9e2XEKg7IsgUCZgE73hEcdnNldmFAZGVi
aWFuLm9yZwAKCRB9e2XEKg7IsrGYAP9BXh/gqdwmKGKOsF/oq/qjroKc1ZIq9grY
Vghv3J0JQQD/c9MmzWF9U2sOOVUoYO2ct5TJhNyGoA9upx8XvQQgxgg=
=Po/t
-----END PGP SIGNATURE-----
```
